invenio_stats/aggregations.py

```diff
@@ -361,17 +395,27 @@ def agg_iter(self, dt):
                     else:
                         aggregation_data[destination] = source(doc, aggregation_data)
 
-                index_name = "stats-{0}-{1}".format(
-                    self.event, interval_date.strftime(self.index_name_suffix)
+                index_name = prefix_index(
+                    "stats-{0}-{1}".format(
+                        self.event, interval_date.strftime(self.index_name_suffix)
+                    )
                 )
 
                 yield {
                     "_id": "{0}-{1}".format(
                         aggregation["key"], interval_date.strftime(self.doc_id_suffix)
                     ),
-                    "_index": prefix_index(index_name),
+                    "_index": index_name,
                     "_source": aggregation_data,
                 }
+                if "parent_recid" in doc:
```

Comment:
disclaimer: I might lack some context here
I find it a little strange that invenio_stats modules "knows" about parent_jd (meaning it depends on rdm data structure), therefore I am not sure if this part should be here

Reply:
This is indeed a bit tricky. It is interestingt that the template files are defined in two different modules (here and in invenio_rdm_records). One of the differences between those two templates are the parent_recid. 

If you have a better idea on how to tackle this, I'd be happy to comply

Reply:
I've removed the references to the parent. Thanks!